test/e2e_node/services/apiserver.go

```diff
@@ -55,7 +56,7 @@ func (a *APIServer) Start() error {
 	errCh := make(chan error)
 	go func() {
 		defer close(errCh)
-		err := apiserver.Run(config)
+		err := apiserver.Run(config, wait.NeverStop)
```

Comment:
We probably have to close the channel in the error case, compare https://github.com/kubernetes/kubernetes/pull/42278/files#diff-ba08be166e7fd2d15ac0e7e725f7e2e7R79.